build.gradle

```diff
@@ -2,7 +2,7 @@ plugins {
   id 'org.sonarqube' version '2.5'
 }
 
-version = '3.1.0-RC7'
+version = '4.0.0-SNAPSHOT'
```

Comment:
use '3.1.0-SNAPSHOT' instead. we should use '4.0.0-SNAPSHOT' after '3.1.0' release.

Reply:
OK, but why this causes the build failure?